xinference/model/llm/transformers/qwen2_vl.py

```diff
@@ -119,6 +119,16 @@ def load(self):
                 torch_dtype="float16",
                 **kwargs,
             ).eval()
+        elif device == "mps":
+            # MacOS special, https://github.com/QwenLM/Qwen2.5-VL/issues/777

```

Comment:
The issue described in issue seems has been fixed, I checked the file: https://huggingface.co/Qwen/Qwen2.5-VL-7B-Instruct/blob/main/preprocessor_config.json

It's correct already.